src/test/java/it/gov/pagopa/payment/controller/BasePaymentControllerIntegrationTest.java

```diff
@@ -317,6 +327,28 @@ private TransactionInProgress checkIfStored(String trxId) {
             Assertions.assertFalse(transactionInProgressRepository.existsById(trxCreated.getId()));
         });
 
+        //useCase 6: trx Status given
```

Comment:
it's too expensive to create a new useCase just to test the getStatus!
remove this!

simply inside method "checkTransactionStored" let's invoke the getStatusTransaction to check the 200 status and the matching content.
Then try invoking again using a mismatched merchantId, verifying the 404